src/Vanilla/Cli/Command/BuildCmd.php

```diff
@@ -49,45 +55,77 @@ public function __construct(Cli $cli) {
      * @inheritdoc
      */
     protected function doRun(Args $args) {
+        $this->getAddonJsonBuildOptions();
+        $this->getBuildOptionsFromArgs($args);
+        $this->validateBuildOptions($args);
+
         $processOptions = [
             'watch' => $args->getOpt('watch') ?: false,
+            'cssTool' => $this->buildConfig['cssTool'],
         ];
 
-        $this->determineBuildProcessVersion($args);
-
         $this->spawnNodeProcessFromPackageMain(
             $this->getBuildProcessDirectory(),
             $processOptions
         );
     }
 
     /**
-     * Determine which build process to use.
+     * Merge in the config values in the addon.json.
+     *
+     * @return void
+     */
+    protected function getAddonJsonBuildOptions() {
+        $addonJsonPath = getcwd().'/addon.json';
+
+        if (file_exists($addonJsonPath)) {
+            $addonJson = json_decode(file_get_contents($addonJsonPath), true);
+
+            // Get the build key and map the old key name
+            if (array_key_exists('build', $addonJson)) {
+                $this->buildConfig = array_merge($this->buildConfig, $addonJson['build']);
+            } else if (array_key_exists('buildProcessVersion', $addonJson)){
+                $this->buildConfig['processVersion'] = $addonJson['buildProcessVersion'];
+            }
```

Comment:
Please do some error handling.

one error if json_decode fails.
one error if build || buildPRocessVersion was not found.

Reply:
if neither is found it falls back to the default (legacy) so I just need to handle the json_decode.